src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp4ij/LanguageServerWrapper.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2020, 2022 Red Hat, Inc.
+ * Copyright (c) 2020, 2023 Red Hat, Inc.
```

Comment:
I think you should leave the Red Hat copyright alone and add a new line saying "(C) Copyright IBM Corp. 2023" or put Copyright first to match Red Hat's.

Reply:
Oops I actually meant this for LanguageServersRegistry.java

Reply:
Resolving this for now, to re-visit once we have more guidance